processImages: True
pathToModel: "/home/rene/catkin_ws/src/background_foreground_segmentation/model.h5"
frameId: "rgb_camera_link"
imageTopic: "/rgb/image_raw"
cameraTopic: "/rgb/camera_info"
segmentationOutTopic: "/rgb/bg_fg_img"
cam:
  height: 1536
  width: 2048
  distortion_model: "rational_polynomial"
  D : [0.44753408432006836, -2.5201666355133057, 0.00042054810910485685, -0.00027887162286788225, 1.4249203205108643, 0.3260822296142578, -2.3430750370025635, 1.3531556129455566]
  K : [977.6222534179688, 0.0, 1022.57177734375, 0.0, 977.51171875, 783.015625, 0.0, 0.0, 1.0]
  R : [1.0, 0.0, 0.0, 0.0, 1.0, 0.0, 0.0, 0.0, 1.0]
  P : [977.6222534179688, 0.0, 1022.57177734375, 0.0, 0.0, 977.51171875, 783.015625, 0.0, 0.0, 0.0, 1.0, 0.0]